source/blog/2016-01-14-atomic-app-0-3-1.html.md

```diff
@@ -0,0 +1,143 @@
+
+The main features of this release are: 
+  * Native persistent storage integration. (Kubernetes only support for now)
+  * HTTPS (TLS) verification and support for OpenShift
+  * OpenShift `stop` added
+  * Nested Nulecule application support for OpenShift
+
+
+READMORE
+
+Bug fixes and changes:
+
+  * Refactor of our OpenShift provider
+  * Compatibility fixes with Atomic CLI and Atomic App integration
+  * CLI restructure to allow params at all levels of interaction
+  * Additional unit tests for OpenShift
+
+__Persistent storage__
+
+The integration of native persistent storage support allows a persistent volume to be defined within the `Nulecule` file rather than the `/artifact` folder of the Nuleculized container. Atomic App will define and create the persistent storage volume claim before proceeding further in deploying the application.
+
+Currently, Kubernetes is the only provider with support for launching a persistent storage volume.
+
+For example, a persistent volume can be claimed within the Nulecule file as follows:
+```yaml
+graph:
+    ...
+    requirements:
+      - persistentVolume:
+          name: "var-log-mongodb"
+          accessMode: "ReadWrite"
+          size: 4
+```
+
+A full example would be:
+```yaml
+---
+specversion: 0.0.2
+id: helloapache-app
+
+metadata:
+  name: Hello Apache App
+  appversion: 0.0.1
+  description: Atomic app for deploying a really basic Apache HTTP server
+graph:
+  - name: helloapache-app
+    params:
+      - name: image
+        description: The webserver image
+        default: centos/httpd
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 80
+    artifacts:
+      docker:
+        - file://artifacts/docker/hello-apache-pod_run
+      kubernetes:
+        - file://artifacts/kubernetes/hello-apache-pod.json
+    requirements:
+      - persistentVolume:
+          name: "var-log-mongodb"
+          accessMode: "ReadWrite"
+          size: 4
+```
+
+And running it:
+```bash
+docker build -t projectatomic/helloapache-ps
+atomic run projectatomic/helloapache-ps --provider=docker
+```
+
+When you deploy your application, Atomic App will automatically take the `persistentVolume` claim under requirements and deploy it to it's respective provider. In the case of Kubernetes, a [persistent volume claim](https://github.com/kubernetes/kubernetes/blob/master/docs/design/persistent-storage.md) will be defined before proceeding in launching a [replication controller](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/replication-controller.md) or [pod](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/pods.md).
+
+__OpenShift stop__
+
+Compatibility for stopping deployed Atomic App containers on OpenShift has now been added.
+
+```
+atomic stop projectatomic/helloapache --provider=openshift /var/lib/atomicapp/yournuleculefile
```

Comment:
we pick up information from the answers.conf.gen file so no need to give `--provider=openshift`

also `yournuleculefile` is probably not good since we actually want them to provide the directory. `yournuleculedir` would be better